routes/index.js

```diff
@@ -1165,11 +1165,14 @@ module.exports = function () {
     const token = req.cookies.jwt
     const from = req.query.from || 0
     const size = req.query.size || 20
+    let topResults;
     let query
     if (req.query.q) {
-      query = `q="${req.query.q}"&size=${size}&from=${from}`
+      query = `q="${req.query.q}"&size=${size}&from=${from}`;
+      topResults = false;
     } else {
-      query = `size=${size}&from=${from}`
+      //query = `size=${size}&from=${from}`
```

Comment:
Let's remove commented code - if you need it, you always can restore it from git